packages/elements-core/src/components/TableOfContents/types.ts

```diff
@@ -29,14 +29,17 @@ export type TableOfContentsGroupItem =
 export type TableOfContentsGroup = {
   title: string;
   items: TableOfContentsGroupItem[];
+  itemsType?: 'article' | 'http_operation' | 'http_webhook' | 'model';
```

Comment:
🤔 does the ToC api need to set this as well?

Reply:
In a way yes. There a `type` property in `TableOfContentsNode` that we'll need to extend with `http_webhook` type